OrbitCore/OrbitProcess.cpp

```diff
@@ -14,7 +14,7 @@
 using orbit_client_protos::FunctionInfo;
 
 Process::Process() {
-  m_ID = 0;
+  process_id_ = 0;
```

Comment:
nit: maybe just `id` ? 
Its a little verbose now: `process.process_id`.

Reply:
also: You are setting this here to `0`; In OrbitProcess.h its set to `-1`.

Reply:
Nice, spot. Renamed and made it also -1 here.